packages/server-wallet/src/engine/engine.ts

```diff
@@ -1010,7 +1006,10 @@ export class SingleThreadedEngine
         tx,
         'approved'
       );
-      this.emit('objectiveStarted', objective);
+
+      // TODO: The response is currently not returned or sent anywhere
```

Comment:
We use a `response` in a couple of our `ChainEventListener`s that never gets sent to anyone. This should get cleaned up when we decouple the chain service from the engine.

Reply:
> Re. [18f62e8](https://github.com/statechannels/statechannels/commit/18f62e8e91f892b1ed36cd7367409ac6b72d1477): what motivated this change?

Meta comment: Do these comments survive a rebase?

Reply:
In particular, I was talking about how we add things to the `response`  (`response.queueCreatedObjective`) but the response is never sent anywhere since this taking place in the `channelFinalized` handler.

About the query: when the `chain-service` is removed from the `engine` this will be refactored and I expect that query to go.

Reply:
Can you commit to removing the query? I don't like adding unnecessary DB queries just to get things to work.

(It's pretty difficult for me to think through "Why does it not matter that this query is added?". It is easier to not think through that if the query will disappear.)